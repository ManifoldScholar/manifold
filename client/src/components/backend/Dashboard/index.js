import Notifications from './Notifications.js';

export default {
  Notifications
};
