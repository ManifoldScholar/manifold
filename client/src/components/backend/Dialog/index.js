import Wrapper from "./Wrapper";
import Confirm from "./Confirm";
import ResetPassword from "./ResetPassword";

export default {
  Wrapper,
  Confirm,
  ResetPassword
};
