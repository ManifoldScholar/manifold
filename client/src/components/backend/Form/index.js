import HasMany from './HasMany';
import Radios from './Radios';
import TextArea from './TextArea';
import TextInput from './TextInput';
import MaskedTextInput from './MaskedTextInput';
import Upload from './Upload';
import Select from './Select';
import Save from './Save';
import Switch from './Switch';
import Date from './Date';
import HigherOrder from './HigherOrder';
import Hidden from './Hidden';
import AvatarBuilder from './AvatarBuilder/AvatarBuilder';
import setter from './setter';
import FieldGroup from './FieldGroup';

export default {
  HasMany,
  Hidden,
  Radios,
  TextArea,
  TextInput,
  MaskedTextInput,
  Upload,
  Select,
  Save,
  Switch,
  Date,
  AvatarBuilder,
  HigherOrder,
  FieldGroup,
  setter
};
