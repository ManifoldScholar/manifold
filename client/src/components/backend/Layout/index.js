import Header from './Header.js';
import PanelNav from './PanelNav';

export default {
  Header,
  PanelNav
};
