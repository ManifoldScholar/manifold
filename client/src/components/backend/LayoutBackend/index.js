import Header from './Header.js';

export default {
  Header
};
