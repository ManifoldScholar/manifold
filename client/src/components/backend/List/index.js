import SimpleList from "./SimpleList";
import Searchable from "./Searchable";

export default {
  SimpleList,
  Searchable
};
