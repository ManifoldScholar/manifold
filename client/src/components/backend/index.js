export Dashboard from './Dashboard';
export Form from './Form';
export Layout from './Layout';
export Project from './Project';
export List from './List';
export Resource from './Resource';
export User from './User';
export Drawer from './Drawer';
export Dialog from './Dialog';
export Navigation from './Navigation';
export Category from './Category';
export Maker from './Maker';
export Event from './Event';
