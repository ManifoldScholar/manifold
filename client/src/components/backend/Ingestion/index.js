import Form from "./Form";
import ConnectionError from "./ConnectionError";

export default {
  Form,
  ConnectionError
};
