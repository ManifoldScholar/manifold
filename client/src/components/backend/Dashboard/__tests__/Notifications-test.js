import React from 'react';
import renderer from 'react-test-renderer';
import { Dashboard } from 'components/backend';

describe("Backend.Dashboard.Notification component", () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <Dashboard.Notifications />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});

