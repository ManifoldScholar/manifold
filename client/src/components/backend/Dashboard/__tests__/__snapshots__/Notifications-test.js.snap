exports[`Dashboard.Notification component renders correctly 1`] = `
<nav
  className="notification-list-block">
  <ul>
    <li>
      <p>
         Notifications are still a 
        <a
          href="https://en.wikipedia.org/wiki/Democracy_in_America">
          work in progress!
        </a>
      </p>
      <button
        className="manicon manicon-x" />
    </li>
    <li>
      <p>
        For now, these are just 
        <a
          href="https://en.wikipedia.org/wiki/Placeholder">
          placeholders
        </a>
      </p>
      <button
        className="manicon manicon-x" />
    </li>
  </ul>
  <a
    className="button-bare-primary"
    onClick={[Function]}
    style={Object {}}>
    Notification History
  </a>
</nav>
`;
